packages/docs/pages/page.tsx

```diff
@@ -0,0 +1,287 @@
+import { H1, Panel, Text } from '@bigcommerce/big-design';
+import { AddIcon, ArrowDropDownIcon } from '@bigcommerce/big-design-icons';
+import { ActionBar, Header, Page } from '@bigcommerce/big-design-patterns';
+import React, { Fragment } from 'react';
+
+import {
+  Code,
+  CodePreview,
+  CodeSnippet,
+  ContentRoutingTabs,
+  GuidelinesTable,
+  List,
+} from '../components';
+import { BackgroundPropsTable, PagePropsTable } from '../PropTables/PagePropTable';
+
+const HeaderPage = () => {
+  return (
+    <>
+      <H1>Page</H1>
+
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          The <Code primary>Page</Code> component serves as a high-level layout component that
+          organizes the content of a page, including optional elements such as a header, message,
+          and action bar. It ensures consistent spacing, background management, and placement of key
+          UI elements.
+        </Text>
+
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>
+            To provide a container for the entire page structure, including header, content,
+            messages, and action buttons.
+          </List.Item>
+          <List.Item>
+            To maintain consistency in page layout and background customization.
+          </List.Item>
+        </List>
+      </Panel>
+
+      <Panel header="Implementation" headerId="implementation">
+        <Fragment key="basic">
+          <Text>
+            To use <Code primary>Page</Code> import the component from the package:
+          </Text>
+          <CodeSnippet>{`import { Page } from '@bigcommerce/big-design-patterns';`}</CodeSnippet>
+        </Fragment>
+        <ContentRoutingTabs
```

Comment:
Could we add an example with the `message` prop?

Reply:
Thanks @chanceaclark , just added the message sample too.